client/src/components/pages/FAQ.tsx

```diff
@@ -8,7 +9,14 @@ const FAQ: React.FC = () => {
   const { t } = useTranslation("translation", { keyPrefix: "faq" });
 
   return (
-    <MarkdownWrapper>
+    <MarkdownWrapper
+      className={css`
```

Comment:
This could be a re-used className! 

```
const pageMarkdownWrapper = css`
  ...
`
```

And then it could be used in FAQ, About, and CookiePolicy